app/assess/templates/assessor_dashboard.html

```diff
@@ -7,7 +7,11 @@
 
 {% block content %}
     <div class="govuk-phase-banner flex-parent-element">
-        <p class="flex-parent-element flexed-element-margins-collapse govuk-!-text-align-right"><a href="{{ sso_logout_url }}" class="govuk-link flexed-element-margins-collapse">Log out</a></p>
+        <p class="flex-parent-element flexed-element-margins-collapse govuk-!-text-align-right">
+            <a href="{{ sso_logout_url }}" class="govuk-link">
+                Log out
+            </a>
+        </p>
```

Comment:
Could we move this into a shared partial? Surely we need this on every page/in the shared header?

Reply:
Have refactored this in https://github.com/communitiesuk/funding-service-design-assessment/pull/117/commits/3afb8ff30a2fef413449550d8902160e57f4576f